### ISSUE TITLE
Difference between Keras' evaluation and accuracy method

### ISSUE THREAD

**Issue by sjhddh** [fchollet/keras#334]
I have made some models using Keras,
However, I find the `evaluation` and `accuracy` method give totally different results, why?

1. using `evaluation`
```python
model.fit(x_train, y_train, batch_size=batch_size, nb_epoch=5, validation_split=0.1, show_accuracy=True)
score = model.evaluate(x_test, y_test, batch_size=batch_size)
print('Test score:', score)
```

2. using `accuracy`
```python
classes = model.predict_classes(x_test, batch_size=batch_size)
acc = np_utils.accuracy(classes, y_test)
print('Test accuracy:', acc)
```

The output of the first one is 69%; but the second one gives 53%.

Am I somewhere wrong?

**Comment by fchollet** [comment]
You are confusing loss and classification accuracy. `evaluate` returns the loss score over the test data (the extract metric depends on the `loss` parameter at compilation time). 

To see both at the same time, use:
```python
loss, acc = model.evaluate(x_test, y_test, show_accuracy=True)
```
